src/Groups/matrices/iso_oscar_gap.jl

```diff
@@ -56,7 +56,20 @@ function ring_iso_oscar_gap(F::T) where T <: Union{Nemo.FqNmodFiniteField, Nemo.
    G = GAP.Globals.GF(Fp_gap, f_gap)
 
    # compute matching bases of both fields
-   Basis_G = GAP.Globals.Basis(G)
+   if GAP.Globals.IsAlgebraicExtension(G)
+#FIXME:
+# As soon as the problem from https://github.com/gap-system/gap/issues/4694
+# is fixed, go back to `Basis_G = GAP.Globals.Basis(G)` also in this case.
```

Comment:
We probably should do more, though, and try to actively avoid this case, e.g. by checking if the Oscar polynomial is a Conway polynomial, in which case we can just use `GAP.Globals.GF(GAP.Obj(p),d)` instead of `GAP.Globals.GF(Fp_gap, f_gap)`.

Reply:
Yes, but this is independent of the current workaround.
I can create a pull request for this improvement.